lib/Models/Catalog/Gtfs/GtfsCatalogItem.ts

```diff
@@ -279,19 +306,44 @@ export default class GtfsCatalogItem extends MappableMixin(
       uri: new ConstantProperty(this.model.url),
       upAxis: new ConstantProperty(this._cesiumUpAxis),
       forwardAxis: new ConstantProperty(this._cesiumForwardAxis),
-      scale: new ConstantProperty(
-        this.model.scale !== undefined ? this.model.scale : 1
-      ),
+      scale: new ConstantProperty(this.model.scale ?? 1),
       heightReference: new ConstantProperty(HeightReference.RELATIVE_TO_GROUND),
       distanceDisplayCondition: new ConstantProperty({
         near: 0.0,
         far: this.model.maximumDistance
-      })
+      }),
+      maximumScale: new ConstantProperty(this.model.maximumScale),
+      minimumPixelSize: new ConstantProperty(this.model.minimumPixelSize ?? 0),
+      shadows: ShadowMode.DISABLED
     };
 
     return new ModelGraphics(options);
   }
 
+  @computed
+  private get _coloredModels() {
```

Comment:
We might want this to return `undefined` when in 2d mode. Right now the map freezes with some LeafletVisualizer errors when switching to 2d.

Reply:
I'll try to debug those LeafletVisualizer errors.

Reply:
The LeafletVisualizer errors are because the image in cross-origin. Maybe billboards in LeafletVisualizer need a cross origin attribute?

Reply:
The LeafletVisualizer errors occur inside `recolorBillboard` when the billboard `color !== Color.WHITE`. This is becuase the image is fetched without `crossOrigin="anonymous"`.
One funny thing is that the layer works in 2d if you change the opacity to 100% (opacity is dealt with through the `color` attribute of the billboard).

Reply:
Fixed